ci/aws/aws-insts.tf

```diff
@@ -70,3 +75,31 @@ resource "aws_instance" "k8s-node" {
     private_key = file(var.private_key_file)
   }
 }
+
+resource "random_integer" "snaps-k8s-secondary-subnet-mid" {
```

Comment:
This range can be larger (min: 0, max: 255)

Reply:
We can deal with this later as the second subnet's NICs aren't currently being attached to the VMs